internal/query/event.go

```diff
@@ -66,7 +66,7 @@ func filterAuditLogRetention(ctx context.Context, auditLogRetention time.Duratio
 	if callTime.IsZero() {
 		callTime = time.Now()
 	}
-	oldestAllowed := callTime.Add(-auditLogRetention)
+	oldestAllowed := callTime.UTC().Add(-auditLogRetention)
```

Comment:
why did you add this?

Reply:
I carefully tested it again, and you are right. I removed it.